packages/spineClient/src/live-spine-client.ts

```diff
@@ -0,0 +1,64 @@
+import {ClientRequest, SpineResponse} from "./models/spine"
```

Comment:
Are the tests for this to be added when it's returning a non-400 response?

Reply:
yes